keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -115,14 +117,15 @@ class SqlDeliveryConfigRepository(
           .toSet()
           .let { artifacts ->
             jooq
-              .select(ENVIRONMENT.UID, ENVIRONMENT.NAME, ENVIRONMENT.CONSTRAINTS)
+              .select(ENVIRONMENT.UID, ENVIRONMENT.NAME, ENVIRONMENT.CONSTRAINTS, ENVIRONMENT.NOTIFICATIONS)
               .from(ENVIRONMENT)
               .where(ENVIRONMENT.DELIVERY_CONFIG_UID.eq(uid))
-              .fetch { (environmentUid, name, constraintsJson) ->
+              .fetch { (environmentUid, name, constraintsJson, notificationsJson) ->
                 Environment(
                   name = name,
                   resources = resourcesForEnvironment(environmentUid),
-                  constraints = mapper.readValue(constraintsJson)
+                  constraints = mapper.readValue(constraintsJson),
+                  notifications = mapper.readValue(notificationsJson ?: "[]")
```

Comment:
You can't [give a default value to a `text` field](https://bugs.mysql.com/bug.php?id=21532), so I'm handling the null case for this field here so it's easy to roll this out.

Reply:
good call